### PR TITLE
Player page

### PR DESCRIPTION
**Pull request by amita-roy**
I have created this Video-Player project in collaboration with my partner.

This is our second repository for this project as our first repository's pull request got merged by mistake after the update for required changes as per the reviewer comments so we were asked to create new repository.

Old repository link: 
[https://github.com/Amita-Roy/video-player-clone/tree/video-landing-page](url)

old pull requests:
[https://github.com/Amita-Roy/video-player-clone/pull/1](url)

last review comments: 
[https://github.com/Amita-Roy/video-player-clone/pull/3](url)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+            <div class="icons flex-sb-center dark-gray">
+              <p class="right-margin">
+                <span><i class="fas fa-thumbs-up"></i></span> 42k
+              </p>
+              <p class="right-margin">
+                <span><i class="fas fa-thumbs-down"></i></span> 1k
+              </p>
+              <p class="right-margin">
+                <span><i class="fas fa-share"></i></span> Share
+              </p>
```
Comment (peterrobert) [inline index.html]: Kindly the share should be a button, not an icon. You can add a button tag
button

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
@Amita-Roy and @ershadul1  **VERY IMPRESSIVE WORK** :+1: 
You have done all the suggested changes :clap: But there is one thing that has prevented me from approving this project. Kindly check the comment below fix and submit the project again

NB: Kindly don't close this pull request until your project is approved use the same link to submit the code review request

**HAPPY CODING** :computer:

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi, @Amita-Roy  and @ershadul1, great job with fixing the comments :+1: I hereby approve the project :slightly_smiling_face: You can go ahead and merge to master, submit the project completion form :+1: 

Good luck with your next projects 
_Happy coding_ :man_technologist:  :woman_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**